### PR TITLE
Setup CI with GitHub Actions

### PR DESCRIPTION
**Pull request by ubruhin**
Based on #3 (needs to be merged first).

### CODE REVIEW

**Comment by dbrgn** [comment]
Cool, thanks! Can you rebase? #3 was merged.

**Comment by ubruhin** [comment]
Rebased to master.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install qt and build tools
+        run: |
+          sudo apt-get update &&
+          sudo apt-get install -y --no-install-recommends \
+            pkg-config g++ make cmake \
+            qt5-default qttools5-dev-tools
+      - name: Run cmake
```
Comment (dbrgn) [inline .github/workflows/ci.yml:51]: I'd add another step before this line:

    - name: Show cmake version
      run: cmake --version

**Review by dbrgn — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    strategy:
+      matrix:
+        runner:
+          - "ubuntu-16.04"
+          - "ubuntu-18.04"
+          - "ubuntu-20.04"
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v1
```
Comment (dbrgn) [inline .github/workflows/ci.yml]: Ah, and you could use checkout@v2 here (and below).

Comment (ubruhin) [reply .github/workflows/ci.yml]: Done.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ubruhin) [reply .github/workflows/ci.yml:51]: Done.

**Review by dbrgn — APPROVED** [review]

**Pull request merged**